pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+
+**Zasada działania**
+: Narzędziami oceny mogą być wtyczki (rozszerzenia) do narzędzi autorskich i przeglądarek internetowych, narzędzia wiersza poleceń, aplikacje komputerowe lub mobilne oraz usługi internetowe. W zależności od procesu projektowania i tworzenia można łączyć różne narzędzia.
+
+**Standardy**
+: Różne organizacje i rządy mogą wymagać zgodności z różnymi standardami dostępności, dlatego różne narzędzia obsługują te standardy. Wytyczne W3C dla dostępności treści internetowych (WCAG) są uznawane na arenie międzynarodowej za standard dostępności stron internetowych i obsługiwane przez większość narzędzi.
+
+**Typ pliku**
+: Większość narzędzi oceny sprawdza dostępność treści HTML. Niektóre sprawdzają inne technologie internetowe, takie jak EPUB, WAI-ARIA, CSS, SVG i PDF.
+
+**Zakres**
+: Zakres tego, co narzędzie oceny może automatycznie testować, różni się w zależności od narzędzia. Niektóre narzędzia sprawdzają pojedynczą stronę, inne natomiast sprawdzają całe grupy powiązanych stron. Niektóre mogą również uzyskać dostęp do treści chronionych hasłem.
+
+**System operacyjny**
+: Niektóre narzędzia działają tylko w określonych systemach operacyjnych, takich jak Windows, MacOS, Linux, Android, iOS.
+
+**Przeglądarka**
+: Niekiedy narzędzia działają tylko w określonych przeglądarkach, takich jak Chrome, Edge, Safari, Firefox itp.
+
+**Deklaracja dostępności**
+: Ważne jest, aby narzędzia oceny były dostępne, aby mogły z nich korzystać osoby z niepełnosprawnościami. Niektórzy dostawcy podają informacje na temat dostępności swoich narzędzi w deklaracji dostępności.
+
+**Zasady ACT**
+: Dostawcy narzędzi mogą opisać, czy i w jaki sposób ich narzędzia obsługują zasady testowania zgodności dostępności. Jeżeli istnieje raport dotyczący wdrożenia zasad, w sekcji szczegółów narzędzia na Liście narzędzi oceny dostępności cyfrowej będzie podane łącze do raportu. Więcej informacji można znaleźć w artykule [[Wdrażanie zasad ACT w narzędziach i metodologiach oceny]](https://www.w3.org/WAI/standards-guidelines/act/implementations/).
+
+**Język**
+: Narzędzia oceny obsługują różne języki. Obejmuje to interfejs użytkownika narzędzia, a także język obsługiwanych treści (na przykład w celu wykrycia barier językowych).
+
+**Wsparcie dla**
+: Niektóre narzędzia oceny dostępności cyfrowej mogą wygenerować pełny raport dostępności. Możesz filtrować, aby znaleźć narzędzia, które mogą Ci pomóc:
+
+  - *Wygenerować raport z wynikami oceny:* Raporty są przydatne do określenia zgodności z kryteriami, które można sprawdzić automatycznie i ręcznie. Może to również obejmować ocenę dostępności.
+
+  - *Wyświetlać informacje w ramach stron internetowych:* Oznacza to, że narzędzie może wstawiać tymczasowe ikony i znaczniki, aby wyświetlić wyniki testów dostępności. Dzięki temu można zobaczyć wyniki testu w kontekście dokumentu. Są one również przydatne do zrozumienia względnego znaczenia każdego problemu.
```

Comment:
@zwiastunsw This part seems to have been slightly reworded:

Original version:
> They are useful to view issues in context. 

Polish translation re-translated into English (may be inaccurate, feel free to rectify):
> This allows you to see the test results in the context of the document.

"Issues" and "test results" have a slightly different meaning. Do you think of an alternative closer to the original meaning?